typescript/infra/config/environments/mainnet3/helloworld.ts

```diff
@@ -20,12 +20,12 @@ export const hyperlane: HelloWorldConfig = {
     namespace: environment,
     runConfig: {
       mode: HelloWorldKathyRunMode.Service,
-      fullCycleTime: 1000 * 60 * 60 * 24, // every 24 hours
+      fullCycleTime: 1000 * 60 * 60 * 24 * 5, // every 5 days, 13 * 12 messages = 156 messages is little less than once an hour
     },
     messageSendTimeout: 1000 * 60 * 8, // 8 min
     messageReceiptTimeout: 1000 * 60 * 20, // 20 min
     connectionType: RpcConsensusType.Fallback,
-    cyclesBetweenEthereumMessages: 3, // Skip 3 cycles of Ethereum, i.e. send/receive Ethereum messages every 32 hours.
+    cyclesBetweenEthereumMessages: 2, // Skip 1 cycle of Ethereum, i.e. send/receive Ethereum messages every 5 days (not great since we still send like 12 in that cycle)
```

Comment:
Kinda weird but I guess we add 1 to this number here https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/main/typescript/infra/scripts/helloworld/kathy.ts#L310 so we may need to change this # to 1? or remove that addition

Reply:
Sorry Yeah should be 1